### PR TITLE
Krishnapro

### PR DESCRIPTION
**Pull request by Krishnapro**
resolve #691

### CODE REVIEW

**Comment by Krishnapro** [comment]
I am can't  able to find where i make a mistake please help me.

**Comment by CodingSpecies** [comment]
@Krishnapro , if I was you, delete your fork, and then refork, then see this
### What to do:

- [x] Take a screenshot of your profile, up to where it says `pinned repos`.
- [x] As you just took a screenshot, just `ctrl v` it into the issue. Or, save it to your computer and then upload it to GitHub, and it will generate a custom image URL, when you copy image address.
- [x] Go to your forked copy of the repo, and go to `data.json` and add your name, paste in the image url and fill out the rest.
### Now, just open a pull request with your changes!  🎉

**Comment by CodingSpecies** [comment]
@Krishnapro If you want to to expand on my answer or have any questions, feel free to ask.

File: data.json [hunk 1]
```diff
@@ -1306,7 +1306,14 @@
     "githubUsername": "BADSHAH-AMAN",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/132057221-2e7c6dc4-a664-4565-9909-101b8fadfa4b.png",
     "issueNumber": 685
-  },
+   },
```
Comment (Panquesito7) [inline data.json:1309]:
```suggestion
  },
```

**Comment by schmelto** [comment]
Resolved in #704

**Pull request closed without merging**